docs/src/upkeep/images.md

```diff
@@ -156,74 +47,62 @@ And finally build the images:
 $ sudo make pxe-all
 ```
 
-### Building the ISO images.
-
-The ISO images do not follow the same build process as the rest of the
-system.  To build these you should use the `build-x86-images.sh`
-script which does a one-shot build of all the images.
-
-First build the x86_64 images:
-
-```
-$ sudo ./build-x86-images.sh -r /hostdir/binpkgs
-```
-
-Then build the musl equivalents:
-
-```
-$ sudo ./build-x86_64-images.sh -a x86_64musl -r /hostdir/binpkgs/musl
-```
+## Collecting the Images
 
-And finally the x86 images:
+Once all images are built, they need to be collected from the Github CI
+artifacts. This can be done via the Github CI web interface, on the "Summary"
+tab of the CI run, or void-mklive's `release.sh` can download them to a
+directory called `void-live-<date>` with:
 
 ```
-$ sudo ./build-x86-images.sh -a i686 -r /hostdir/binpkgs
+$ ./release.sh dl
 ```
 
-## Collecting the Images
+> Note: this currently assumes latest successful CI run is the one to download.
 
-Once all images are built, they need to be collected into a directory,
-signed, and moved into place.
+Once downloaded, verify all sha256sums match:
 
 ```
-$ mkdir <date>
-$ mv void*<date>* <date>/
+$ cd void-live-<date>
+$ sha256sum -c sha256sum.txt
 ```
 
-Generate the sha256sums:
-
-```
-$ cd <date>
-$ sha256 * > sha256.txt
-```
+The images can then be uploaded to [DevSpace](/services/devspace.md) or the
+mirrors for testing.
 
 ## Signing the Images
 
-Signing the images is done after all the images have been checked and
-validated, and after the decision has been made to promote the set to
-`current`.
+Signing the images is done after all the images have been checked and validated,
+and after the decision has been made to promote the set to `current`.
 
 Generate a new signing key:
 
 ```
-$ pwgen -cny 25 1 > void-release-<date>.key
-$ signify -G -p void-release-<date>.pub -s void-release-<date>.sec -c "This key is only valid for images with date <date>."
+$ export DATECODE=<date>
+$ pwgen -cny 25 1 > void-release-$DATECODE.key
+$ minisign -G -p void-release-$DATECODE.pub -s void-release-$DATECODE.sec -c "This key is only valid for images with date $DATECODE."
 ```
 
 Copy the public half of this key to the void-release-keys package in
-void-packages and make a release.  Copy all key material to the
-owner's team in keybase and ensure that the copy has been completed
-successfully.
+void-packages and make a release. Copy all key material to the owner's team in
+keybase and ensure that the copy has been completed successfully.
 
-Copy the `sha256.txt` file to your local workstation and sign it with the appropriate key.
+Copy the `sha256sum.txt` file to your local workstation and sign it with the
+appropriate key.
 
 ```
 $ signify -S -e -s void-release-<date>.sec -m sha256.txt -x sha256sum.sig
 ```
```

Comment:
@the-maldridge can you provide the exact process you used for generating and signing here?

Reply:
my shell history got clobbered, but I specified the commands as shown in the help, so -S -x <sig> -s <sec> -c <date comment> -t <date comment> -m sha256sum.sig